include/chemfiles/formats/MMTF.hpp

```diff
@@ -21,20 +21,24 @@ class MMTFFormat final: public Format {
 public:
     MMTFFormat(std::string path, File::Mode mode, File::Compression compression);
 
-    ~MMTFFormat() override = default;
+    ~MMTFFormat() override;
     MMTFFormat(const MMTFFormat&) = delete;
     MMTFFormat& operator=(const MMTFFormat&) = delete;
     MMTFFormat(MMTFFormat&&) = default;
     MMTFFormat& operator=(MMTFFormat&&) = default;
 
     void read_step(size_t step, Frame& frame) override;
     void read(Frame& frame) override;
+    void write(const Frame& frame) override;
     size_t nsteps() override;
 
 private:
 
     mmtf::StructureData structure_;
 
+    std::string filename_;
+    size_t writeAtomLoc_ = 0;
```

Comment:
What does is this used for ? It seems to be used as a flag to know whether to write or not in the destructor, but the variable name seems to indicate something else.

Reply:
Could you add a bit of doc for this and the other members ?

Reply:
Sure thing.